meson.build

```diff
@@ -1,6 +1,6 @@
 project('gtuber', 'c',
   version: '0.0.1',
-  meson_version: '>=0.53.0',
+  meson_version: '>=0.58.0',
```

Comment:
What required a version bump? This is a question only.

Reply:
meson.global_source_root() & str.replace()